arch/arm64/boot/dts/rockchip/overlays/Makefile

```
@@ -431,7 +431,8 @@ dtb-$(CONFIG_CLK_RK3588) += \
 	rock-5b-radxa-display-10hd.dtbo \
 	rock-5b-radxa-display-8hd.dtbo \
 	rock-5b-rpi-camera-v2.dtbo \
-	rock-5b-sata.dtbo
+	rock-5b-sata.dtbo \
+	rk3588-enable-mali.dtbo
```

```suggestion
	rk3588-mali-enable.dtbo
```
倒装语序在文件排序时更容易把相同类型的overlay放到一起